pom.xml

```diff
@@ -2067,6 +2067,7 @@
         <module>cdap-test</module>
         <module>cdap-integration-test</module>
         <module>cdap-ui</module>
+        <module>cdap-formats</module>
```

Comment:
Move it to higher position, right after `cdap-api`, since the only inter-module dependency it has is only `cdap-api`.